Cargo.toml

```diff
@@ -19,8 +19,8 @@ rust-version = "1.76.0"
 # Don't increase beyond what Firefox is currently using: https://searchfox.org/mozilla-central/source/Cargo.lock
 libc = { version = "0.2", default-features = false }
 
-[dev-dependencies]
-rand = { version = "0.8", default-features = false, features = ["std", "std_rng"] }
+[target.'cfg(target_os = "linux")'.dependencies]
+neli = { version = "0.6", default-features = false }
```

Comment:
Adding `neli` increases the dependency tree of `mtu` quite a bit:

```
mtu v0.2.0 (/home/mxinden/code/github.com/mozilla/mtu)
├── libc v0.2.158
└── neli v0.6.4
    ├── byteorder v1.5.0
    ├── libc v0.2.158
    ├── log v0.4.22
    └── neli-proc-macros v0.1.3 (proc-macro)
        ├── either v1.13.0
        ├── proc-macro2 v1.0.86
        │   └── unicode-ident v1.0.12
        ├── quote v1.0.37
        │   └── proc-macro2 v1.0.86 (*)
        ├── serde v1.0.209
        │   └── serde_derive v1.0.209 (proc-macro)
        │       ├── proc-macro2 v1.0.86 (*)
        │       ├── quote v1.0.37 (*)
        │       └── syn v2.0.77
        │           ├── proc-macro2 v1.0.86 (*)
        │           ├── quote v1.0.37 (*)
        │           └── unicode-ident v1.0.12
        └── syn v1.0.109
            ├── proc-macro2 v1.0.86 (*)
            ├── quote v1.0.37 (*)
            └── unicode-ident v1.0.12
```

That said, most of these are already audited:

https://searchfox.org/mozilla-central/source/supply-chain/audits.toml

Reply:
I'd be fully OK with trying to code against the kernel API directly. Let me see how difficult that would be.